helpers_test.go

```diff
@@ -64,6 +64,16 @@ func TestValidateTargetTable(t *testing.T) {
 		require.Equal(t, "t_1_2", validTable.Name())
 	})
 
+	t.Run("valid table name enclosing", func(t *testing.T) {
```

Comment:
looks good. maybe adding a test for unmatched and mismatched char would be nice if that is not already covered elsewhere. e.g., `"[t_1_2"` or `"[t_1_2)`